src/includes/getting-started-primer/apple.mdx

```diff
@@ -8,6 +8,12 @@
 
 **Features:**
 
+- Multiple types of errors are captured, including:
+  - C++ exceptions
+  - Mach exceptions
+  - Unhandled exceptions
+  - UNIX signal
+  - Out of memory
```

Comment:
Strictly speaking, I think it's not correct, we only capture breadcrumbs and optionally an event

Reply:
But since I learned that everyone else is guessing when an OOM crash happened, I think we are fine merging it.

Reply:
We added a feature in Q1 to create an event for OOM (it's on by default): https://github.com/getsentry/sentry-cocoa/pull/1015

It's based on heuristics, but as you said, so is anyone else doing this since it's the only way